oci-hooks/qm-device-manager/README.md

```diff
@@ -4,9 +4,11 @@ The QM Device Manager OCI Hook provides dynamic device access management for QM
 
 ## Overview
 
-The device manager hook allows containers to request specific device access through annotations. The hook dynamically discovers and mounts the requested devices at container creation time.
+The device manager hook allows containers to request specific device access through annotations. The hook dynamically discovers and mounts the requested devices at container creation time. It supports both device categories and Wayland seat-based device access.
 
-## Supported Devices
+## Supported Device Types
+
+### Traditional Device Categories
```

Comment:
I think there should be more info related devices injections and how they are exposed to QM, 

I assume something explains that devices are injected to json hook file on precreate step of next qm start 

linux.resources.devices[] – allows access to the device 

```
  "resources": {
    "devices": [
      {
        "allow": true,
        "type": "c",
        "major": 10,
        "minor": 232,
        "access": "rwm"
      }
      // ... other resource rules
    ]
  }
}
```


I would also advice to point to the relevant pytest verifies the discovery and injection